### DISCUSSION TITLE
Welcome Center - Questions about doors and chests.

### DISCUSSION THREAD

**Discussion by meggggan** [GriefPrevention/GriefPrevention#1199] [Q&A] [answered]
I would like to make a welcome center for all new users.  I plan on a small building with a chest or two with golden shovels and sticks for claiming.  I have the default settings on, I had someone test and they couldn't open the wooden door or chest even after the claim on land was removed. 

1. I thought that doors (except iron doors) are able to be opened regardless of claim with the default setting.
2. Are objects built on previously claimed land (chests included) not able to be opened even after the claim is removed?
3. What is the proper way to go about this? 

Thanks!
Megan

**Comment by Jikoo** [comment, marked as answer]
You probably accidentally left vanilla's spawn protection enabled. This will prevent all interaction (not just building!) within its radius. Check `spawn-protection` in your server.properties.

**Reply by meggggan** [reply]
Will do.  Thanks!
